mint.json

```
@@ -42,18 +42,38 @@
       "group": "Home",
       "pages": [
         "welcome",
-        "get-started",
-        "philosophy"
+        "quickstart",
+        "philosophy",
+        "feedback"
       ]
     },
     {
       "group": "Create and collect",
       "pages": [
         "overview",
-        "local-upload",
+        "csv-upload",
         "widget",
         "public-portal",
         "chrome-extension",
+        "export",
+        "review-spam"
+      ]
+    },
+    {
+      "group": "Feedback",
+      "pages": [
+        "schema",
+        "search"
+      ]
+    },
+    {
+      "group": "Customers",
+      "pages": [
+        "overview"      ]
```

This is why the nav was broken, this is just pointing to the wrong page file.

```suggestion
      "pages": [
        "customer-overview"
      ]
```